resource/conf.go

```diff
@@ -8,8 +8,35 @@ type Conf struct {
 	// no default page size is set resulting in no pagination if no `limit` parameter
 	// is provided.
 	PaginationDefaultLimit int
+	// ForceTotal controls how total number of items on list request is computed.
+	// By default (User), if the total cannot be computed by the storage andler for
+	// free, no total metadata is returned until the user explicitely request it
+	// using the total=1 query-string parameter. Note that if the storage cannot
+	// compute the total and does not implement the resource.Counter interface,
+	// a "not implemented" error is returned.
+	//
+	// The Always mode always force the computation of the total (make sure the
+	// storage either compute the total on Find or implement the resource.Counter
+	// interface.
+	//
+	// Denied prevents the user from requesting the total.
+	ForceTotal ForceTotalMode
 }
 
+// ForceTotalMode defines Conf.ForceTotal modes
```

Comment:
I agree. I would prefer to fix all comments in a separate PR. Do you know some refact tools to help us with that?